proto/cel/expr/conformance/proto2/test_all_types.proto

```diff
@@ -319,6 +319,25 @@ message TestAllTypes {
     optional string single_name = 405;
   }
 
+  // Formerly defined as reserved field names.
```

Comment:
The Protobuf language also knows [reserved fields](https://protobuf.dev/reference/protobuf/proto3-spec/#reserved). 

Should we clarify, e.g.?

```suggestion
  // Field names formerly defined as reserved CEL identifiers.
```